packages/yarnpkg-core/sources/httpUtils.ts

```diff
@@ -92,7 +93,14 @@ export async function request(target: string, body: Body, {configuration, header
 
   for (const [glob, path] of configuration.get(`caFilePath`)) {
     if (micromatch.isMatch(url.hostname, glob)) {
-      extraHttpsOptions.certificateAuthority = await xfs.readFilePromise(path);
+      let entry = certCache.get(path);
+
+      if (!entry) {
+        entry = await xfs.readFilePromise(path);
```

Comment:
```suggestion
        entry = xfs.readFilePromise(path);
```

Reply:
I had just removed that few seconds ago, was an oversight :)